api/v1/helpers.go

```diff
@@ -73,10 +73,12 @@ func MakeVDB() *VerticaDB {
 			Kind:       VerticaDBKind,
 		},
 		ObjectMeta: metav1.ObjectMeta{
-			Name:        nm.Name,
-			Namespace:   nm.Namespace,
-			UID:         "abcdef-ghi",
-			Annotations: make(map[string]string),
+			Name:      nm.Name,
+			Namespace: nm.Namespace,
+			UID:       "abcdef-ghi",
+			Annotations: map[string]string{
+				vmeta.VClusterOpsAnnotation: vmeta.VClusterOpsAnnotationFalse,
```

Comment:
Cool! This change keeps our unit tests aware that it still relies on admintool-based deployments, which can help avoid a bunch of failures defined in vcluster-based deployments.